tests/catalog/test_hive.py

```diff
@@ -1233,3 +1233,84 @@ def test_create_hive_client_failure() -> None:
         with pytest.raises(Exception, match="Connection failed"):
             HiveCatalog._create_hive_client(properties)
         assert mock_hive_client.call_count == 2
+
+def test_list_views(hive_table: HiveTable) -> None:
```

Comment:
This test sets specific parameters then test those parameters.
I think creating an integration test using the HMS would be useful. That way we can test the actual parameter for HMS